libs/community/langchain_community/tools/tavily_search/tool.py

```diff
@@ -63,6 +63,29 @@ class TavilySearchResults(BaseTool):
r delicious recipes; photos and step-by-step instructions help to make each recipe foolproof. Thanks, Jenn!\nBest steak I've ever made\nHello Chef,\nThank you for those quick tips! My rib eye steaks came out delicious… I am serving it with broccoli and cheese, and some Spanish rice…\nHi my name is Rob. I have always been looking for a way to sear and cook steaks on the stove but Oh my god this worked so well it was better than the grill. I'm wondering if I can use the same method to make Hamburgrrs ? I also took the juices with the oil and made a dipping sauce. I can't wait to try again.\nGlad you enjoyed, Robert! It will work with burgers, too.\nI'm on a low carb diet with a low grocery budget, a combination that rendered my take on this recipe to be done with top round instead of a strip or filet. I have to say, adjusting a minute less per side for the thinner cut, the method described here was absolutely perfect. I have found the one I'm sticking with after some less than stellar results from recipes on other sites. Turned out flavorful, perfectly medium-rare, and-and this was a nice bonus for going with a cut of round-delightfully tender. Thank you so much for the great information!\nThis steak is on my “next week” docket, although obsessing and looking forward begins today 🙂 Would be interested in hearing if you pan sear steaks directly from the fridge, or if you let them come to room temperature first.\nHi Jeff, you definitely can take them out about 30 minutes ahead of time. (I usually forget though and they still turn out well.) 🙂\nCan confirm – this is the *best* way to make steaks. Simple, straightforward, and flawless each time. You can also include slightly more involved rubs, but don't go too crazy and be sure to be careful in what you choose. The very high heat can scald some herbs, so make sure you choose a more heat-tolerant rub or seasoning. The herb butter process here is truly *chefs kiss*\nA great recipe and the only way we prepare steaks! A suggestion for those concerned about the smoke and potential splatter indoors is to heat your skillet and perform the entire operation on your outdoor grill. Works just as well without chancing the smoke alarms!\nAdd a Comment Cancel reply\nYour comment *\nRate the recipe: 5 stars means you loved it, 1 star means you really disliked it\nYour name *\nYour email (will not be published) *\nSave my name, email, and website in this browser for the next time I comment.\nΔ\nThis site uses Akismet to reduce spam. Learn how your comment data is processed.\nWelcome - I'm Jenn Segal - Classically Trained Chef, Cookbook Author & Busy Mom\nOnce upon a time, I went to culinary school and worked in fancy restaurants. Now, I'm cooking for my family and sharing all my tested & perfected recipes with you here! Read more…\nFall Recipes\nPopular Recipes\n22 Quick and Easy Recipes in 30 Minutes (or less)\nPlus 5 Chef Secrets To Make You A Better Cook!\nCooking Question?\nΔ\nAs Featured On\nBuy Now\nBuy Now"}],
```

Comment:
wonder if we can truncate this

Reply:
I feel like the artifact will always be slightly large - but if you think it is important, I can try to find something with a shorter output. Or we can just not show the whole artifact? Lmk what you think